spec/features/user_can_create_glossary_entries_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'sign_up_helper'
+require 'create_glossary_entry_helper'
+
+feature 'Glossary creation' do
+  context 'user can create glossary term' do
+    let(:user) { build(:user) }
```

Comment:
We're not testing sign up here and it takes longer to sign up via the UI than it does via the backend. It's quicker to `let(:user) { create(:user) }` and then simply use the `sign_in` helper ([docs here](https://github.com/heartcombo/devise#integration-tests)).